Formula/execline.rb

```diff
@@ -17,7 +17,6 @@ class Execline < Formula
   def install
     system "./configure",
       "--prefix=#{prefix}",
-      "--disable-static",
```

Comment:
@remko It sounds like this should be told to build shared libraries somehow given that you need a runtime dependency on it anyway.